main.go

```diff
@@ -181,6 +186,18 @@ func mainErr(args []string) error {
 		os.Setenv("GARBLE_DIR", wd)
 		os.Setenv("GARBLE_LITERALS", fmt.Sprint(garbleLiterals))
 
+		if debugDir != "" {
+			if !filepath.IsAbs(debugDir) {
+				debugDir = filepath.Join(wd, debugDir)
```

Comment:
why do you need it to be absolute? I think it should work anyway

Reply:
I thought so to but it doesn't because the path is not relative to the directory we run garble in but relative to every package which is obfuscated. So you get multiple ./test directories under every packe with only the files for that package.

Reply:
Ah, gotcha.